<launch>
	<!-- global rosparam -->
	<param name="use_sim_time" value="true"/>

	<!-- rosbag -->
	<!-- <node pkg="rosbag" type="play" name="player" args="&#45;&#45;clock /home/amsl/ozaki/bagfiles/mocap/0119_infant/2019&#45;01&#45;19&#45;18&#45;26&#45;15.bag &#45;s 60"/> -->
	<node pkg="rosbag" type="play" name="player" args="--clock /home/amsl/ozaki/bagfiles/infant/indoor_dkan/2019-04-04-14-27-47.bag"/>
	<!-- <node pkg="rosbag" type="play" name="player" args="&#45;&#45;clock /home/amsl/ozaki/bagfiles/infant/indoor_dkan/2019&#45;04&#45;04&#45;14&#45;29&#45;45.bag"/> -->

	<!-- rviz -->
	<!-- <node pkg="rviz" type="rviz" name="rviz" args="&#45;d $(find gaussian_sphere_slam)/rviz_config/d_gaussian_sphere.rviz"/> -->

	<!-- sensor driver -->
	<include file="$(find velodyne_pointcloud)/launch/32e_points.launch"/>

	<!-- semi-driver -->
	<node pkg="normal_estimation_pcl" name="omp_flexible_search_range" type="omp_flexible_search_range" output="screen">
		<param name="skip" type="int" value="3"/>
		<param name="search_radius_ratio" type="double" value="0.09"/>
	</node>
	<!-- <node pkg="gaussian_sphere_slam" name="d_gaussian_sphere" type="d_gaussian_sphere" output="screen"> -->
	<!-- 	<param name="skip" type="int" value="3"/> -->
	<!-- 	<!&#45;&#45; <param name="mode_remove_ground" type="bool" value="true"/> &#45;&#45;> -->
	<!-- 	<param name="mode_open_viewer" type="bool" value="false"/> -->
	<!-- 	<param name="mode_clustering" type="bool" value="false"/> -->
	<!-- </node> -->

	<!-- main -->
	<node pkg="clustering_pcl" name="conditional_euclidean_clustering" type="conditional_euclidean_clustering" output="screen">
		<param name="cluster_tolerance" type="double" value="0.1"/>
		<param name="min_cluster_size" type="int" value="100"/>
	</node>
</launch>
